webapp/channels/src/components/latex_inline/latex_inline.test.tsx

```diff
@@ -1,21 +1,28 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {shallow} from 'enzyme';
+import {render} from '@testing-library/react';
```

Comment:
we should be using renderWithContext our own wrapper of testing-library

Reply:
Done